torch/nn/modules/loss.py

```diff
@@ -671,13 +671,13 @@ class BCEWithLogitsLoss(_Loss):
         >>> criterion = torch.nn.BCEWithLogitsLoss(pos_weight=pos_weight)
         >>> criterion(output, target)  # -log(sigmoid(1.5))
         tensor(0.20...)
-        
+
     In the above example, the `pos_weight` tensor's elements correspond to the 64 distinct classes
     in a multi-label binary classification scenario. Each element in `pos_weight` is designed to adjust the
```

Comment:
Not your fault as this is wrong on L663 as well, but we should use double backticks for rendering

```suggestion
    In the above example, the ``pos_weight`` tensor's elements correspond to the 64 distinct classes
    in a multi-label binary classification scenario. Each element in ``pos_weight`` is designed to adjust the
```

Could you fix L663 as well please :)

Reply:
Sure...Just updated it :)